.github/workflows/deploy-prodbox.yml

```diff
@@ -3,6 +3,15 @@ name: Deploy Prodbox
 on:
   workflow_dispatch:
     inputs:
+      regions:
+        description: "Regions to deploy to"
+        required: true
+        default: "us-central1"
+        type: choice
+        options:
+          - "us-central1"
+          - "europe-west1"
+          - "all"
```

Comment:
how will "all" work if there are region-specific secrets?

Reply:
So for now environment variables that are passed at build time aren't the right ones, that'll be my next task https://github.com/dust-tt/tasks/issues/1811.

For secrets, since they are loaded once the container is up and running this is directly handled by the helm deployment

Reply:
but I mean, if I pass "all" as option, and I have ENV_VAR_A specific to region, what will be the value on prodbox?

Reply:
So there are two things: environment variables passed at build time and secrets that are loaded by the deployment.

The former (basically args that are passed in cloudbuild) is not handled yet. While the latter doesn't have any impact on the Docker image itself as it is loaded by the deployment.

Does that make sense? Happy to elaborate a bit more if unclear